src/Core/src/Platform/iOS/TabbedViewExtensions.cs

```diff
@@ -118,5 +119,40 @@ internal static void UpdateiOS15TabBarAppearance(
 			// Set the TabBarAppearance
 			tabBar.StandardAppearance = tabBar.ScrollEdgeAppearance = _tabBarAppearance;
 		}
+
+		internal static UIImage? AutoResizeTabBarImage(UITraitCollection traitCollection, UIImage image)
+		{
+			if (image == null)
+			{
+				return null;
+			}
+
+			CGSize newSize = image.Size;
+
+			//https://developer.apple.com/design/human-interface-guidelines/tab-bars#Target-dimensions
+			bool isRegularTabBar = traitCollection.VerticalSizeClass == UIUserInterfaceSizeClass.Regular;
+			if (image.Size.Width > image.Size.Height) //Wide
+			{
+				newSize.Width = isRegularTabBar ? 31 : 23;
```

Comment:
Can include a comment like "Define size constants according to Apple's guidelines" and a link to documentation?